ccd-definition/CaseEventToComplexTypes/manage-documents/deletedCourtDocument.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CourtAdminDocument",
+    "CaseEventID": "uploadStandardDirections",
```

Comment:
Confusing it's in single file. Those are separate events, we usually configure them in separate files, what do you think?

Reply:
Yep we should 👍